src/lib/structures/game/Game.ts

```diff
@@ -157,7 +167,7 @@ export abstract class Game {
 			const promises: Promise<Message>[] = [];
 
 			for (const player of this.players.values()) {
-				const songsGuessedCorrectly = this.leaderboard.get(player.id) ?? 0;
+				const playersScore = this.leaderboard.get(player.id) ?? 0;
```

Comment:
This variable name is only correct with a ', which you obviously can't put in variable names, so I'd just avoid the conflict altogether.
```suggestion
				const score = this.leaderboard.get(player.id) ?? 0;
```